libs/vkd3d-shader/spirv.c

```diff
@@ -5187,7 +5216,7 @@ static const struct vkd3d_shader_global_binding *vkd3d_dxbc_compiler_get_global_
 
     vkd3d_dxbc_compiler_emit_descriptor_binding(compiler, var_id, binding_info);
 
-    if (flags & VKD3D_SHADER_GLOBAL_BINDING_WRITE_ONLY)
+    if ((flags & VKD3D_SHADER_GLOBAL_BINDING_WRITE_ONLY) && !(flags & VKD3D_SHADER_GLOBAL_BINDING_RAW_SSBO))
         vkd3d_spirv_build_op_decorate(builder, var_id, SpvDecorationNonReadable, NULL, 0);
```

Comment:
Should decorate SRV SSBO as NonWritable here on var_id.

Reply:
That is (again) not what glslang does, which decorates the struct member. We're already doing that.

Reply:
Ok.